Makefile

```diff
@@ -4,7 +4,8 @@ IMAGE_NAME:=codice/ddf-base
 GIT_SHA:=$(shell git rev-parse HEAD)
 MASTER_SHA:=$(shell git show-ref -s refs/heads/master)
 ifneq (${MASTER_SHA}, ${GIT_SHA})
-	IMAGE_VERSION=${GIT_SHA}
+    IMAGE_VERSION=2.27-alpine 	
```

Comment:
❔ Does this mean we would overwrite this and actually have different images with the same version?

Reply:
I think we would be better off merging this to a new branch (i.e. 2.27-alpine).
And then also just removing that if block and replacing with `IMAGE_VERSION=2.27-alpine`.

Reply:
@frnkshin do you think we would make changes on the branch? If not just tagging and changing master would work too right?